integrationtests/src/test/java/io/kroxylicious/proxy/KroxyConfigBuilder.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.dataformat.yaml.YAMLGenerator;
+
+public class KroxyConfigBuilder {
+
+    public static final ObjectMapper OBJECT_MAPPER = new ObjectMapper(new YAMLFactory().disable(YAMLGenerator.Feature.WRITE_DOC_START_MARKER));
+
+    public record Proxy(String address,
+                        @JsonInclude(JsonInclude.Include.NON_NULL) String keyStoreFile,
+                        @JsonInclude(JsonInclude.Include.NON_NULL) String keyPassword) {
+    }
+
+    public record Cluster(@JsonGetter("bootstrap_servers") String bootstrapServers) {
+    }
+
+    public record Filter(String type, @JsonInclude(JsonInclude.Include.NON_EMPTY) Map<String, Object> config) {
+    }
+
+    private Proxy proxy;
+    private final Map<String, Cluster> clusters = new LinkedHashMap<>();
+    private final List<Filter> filters = new ArrayList<>();
+
+    public KroxyConfigBuilder(String proxyAddress) {
+        proxy = new Proxy(proxyAddress, null, null);
+    }
+
+    public KroxyConfigBuilder withDemoCluster(String bootstrapServers) {
+        clusters.put("demo", new Cluster(bootstrapServers));
```

Comment:
nit: `withDemoCluster` is not a clear name for me. Maybe `withSimpleCluster` or `withDefaultCluster`?

Reply:
WE should probably stop using the `demo` name and give the default cluster a better name.